### ISSUE TITLE
Trade Form Ignores Duplicate Cards

### ISSUE THREAD

**Issue by alicelmalice** [filler00/etcg#12]
I accidentally sent a trade request to someone requesting two copies of the same card. It somehow went through but it only grabbed one copy of the card. I didn't want both copies but the trade form accepted it anyway and only grabbed the one card into pending.

Does this make sense? orz

**Comment by tooblue** [comment]
Yup, makes sense! Will investigate and fix for the next release. Thanks! :heart:
